app/controllers/application_controller.rb

```diff
@@ -319,6 +319,10 @@ def prompt_to_verify_mfa
     redirect_to user_two_factor_authentication_url
   end
 
+  def go_to_confirmation_page
+    auth_method_confirmation_url
+  end
```

Comment:
this method is marked as not covered by code coverage, and I'm not seeing it used anywhere in the diff, can we remove it?